virtualcluster/pkg/syncer/resources/configmap/checker.go

```diff
@@ -63,6 +64,13 @@ func (c *controller) PatrollerDo() {
 	}
 	pSet := differ.NewDiffSet()
 	for _, pCM := range pConfigMaps {
+		// Ingore both RootCACertConfigMapName and TenantRootCACertConfigMapName from the super.
```

Comment:
Question: Should we ignore or compare against the right configMap stored in in vc namespace? There still has a change the configmap in super is altered unexpectedly? (You can make a follow up change if you agree.)

Reply:
You are right, that is a miss in the current setup. Let me think about how to handle this and I will do a follow up.

Reply:
Some combination of maybe a label on the `tenant-kube-root-ca.crt` that maps the name back to the `kube-root-ca.crt` and checks isn't contents. I think the one thing we do have going for us is right now when this is enabled we still reconcile the tenants `kube-root-ca.crt` so each time checker runs it pops an update back on the reconciler queue which will try to update the super when there is a difference so I think we actually do cover this but in a semi inefficient way.